vue/templates/_package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "bs-todo",
+  "name": "hello-blockstack",
   "version": "6.0.1",
   "description": "A decentralized Todo app built on Blockstack",
   "author": "Michael Stivala <michaelstivala@me.com> (https://michaelstivala.com/)",
```

Comment:
We should probably change the author line as well

Reply:
done.